crates/rattler_virtual_packages/src/lib.rs

```diff
@@ -130,11 +176,21 @@ impl From<VirtualPackage> for GenericVirtualPackage {
 impl VirtualPackage {
     /// Returns virtual packages detected for the current system or an error if the versions could
     /// not be properly detected.
-    pub fn current() -> Result<&'static [Self], DetectVirtualPackageError> {
-        static DETECTED_VIRTUAL_PACKAGES: OnceCell<Vec<VirtualPackage>> = OnceCell::new();
-        DETECTED_VIRTUAL_PACKAGES
-            .get_or_try_init(try_detect_virtual_packages)
-            .map(Vec::as_slice)
+    #[deprecated(since = "1.0.4", note = "Use `Self::detect` instead")]
+    pub fn current() -> Result<Vec<Self>, DetectVirtualPackageError> {
+        try_detect_virtual_packages_with_overrides(&VirtualPackageOverrides::none())
+    }
+
+    /// Detect the virtual packages of the current system with the default overrides.
+    pub fn detect() -> Result<Vec<Self>, DetectVirtualPackageError> {
```

Comment:
Lets just remove this function in favor of `detect_with_overrides`. I dont think we need an overload without an argument.

Reply:
I think
`let virtual_packages = rattler_virtual_packages::VirtualPackage::detect_with_overrides(&rattler_virtual_packages::VirtualPackageOverrides::default())?;`
is not very concise :sweat_smile:

Reply:
`VirtualPackage::detect(&Default::default())` seems fine to me.

Reply:
hmm, yeah, that works. Somehow, `Default` is more robust than what I imagined.